native/Avalonia.Native/src/OSX/AvnWindow.mm

```diff
@@ -384,16 +384,14 @@ - (BOOL)windowShouldZoom:(NSWindow *_Nonnull)window toFrame:(NSRect)newFrame
     return true;
 }
 
--(void)resignKeyWindow
+-(void)windowDidResignKey:(NSNotification *)notification
 {
     if(_parent)
         _parent->BaseEvents->Deactivated();
 
     [self showAppMenuOnly];
     
     [self invalidateShadow];
-
-    [super resignKeyWindow];
```

Comment:
Are you sure that we don't need to call `[super windowDidResignKey: notification]` here?

Reply:
its an optional member... that doesnt compile.